x-pack/plugins/session_view/common/types/process_tree/index.ts

```diff
@@ -33,27 +33,37 @@ export interface ProcessEventResults {
   events: any[];
 }
 
+export interface Teletype {
+  descriptor: number;
+  type: string;
+  char_device: {
+    major: number;
+    minor: number;
+  };
+}
+
 export interface ProcessFields {
+  entity_id: string;
   args: string[];
   args_count: number;
-  entity_id: string;
+  command_line: string;
   executable: string;
-  interactive: boolean;
   name: string;
+  interactive: boolean;
   working_directory: string;
   pid: number;
-  pgid: number;
-  user: User;
   start: Date;
   end?: Date;
+  user: User;
   exit_code?: number;
+  tty: Teletype;
 }
 
 export interface ProcessSelf extends ProcessFields {
-  parent: ProcessFields;
-  session: ProcessFields;
-  entry: ProcessFields;
-  last_user_entered?: ProcessFields;
+  parent: ProcessSelf;
```

Comment:
shouldn't this be ProcessFields? process.parent should not have session_leader, entry_leader. 

The anemic group_leader is a property of process.parent, but we can omit this since sesison view doesn't necessary need the info. yet.